Misc/NEWS.d/next/Library/2020-02-16-17-38-33.bpo-35885.VijyzZ.rst

```diff
@@ -0,0 +1,3 @@
+:class:`configparser` :func:`write()` method have a new parameter `indent` of type string.
+When indent is set, the parser will indent the lines according to the given input value.

```

Comment:
IMO this is too much information for the NEWS entry, but something like this should be added to the doc-string.